boards/spracing/h7extreme/init/rc.board_defaults

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+#
+# SP Racing H7 EXTREME specific board defaults
+#------------------------------------------------------------------------------
+# system_power unavailable
+param set CBRK_SUPPLY_CHK 894281
+
+if param compare SYS_AUTOSTART 0
+then
+	# Select the Generic 250 Racer by default
+	param set SYS_AUTOSTART 4050
+	set AUTOCNF yes
+fi
+
+set LOGGER_BUF 64
+
+if [ $AUTOCNF = yes ]
+then
+	# Disable safety switch by default
+	param set CBRK_IO_SAFETY 22027
+
+	# use the Q attitude estimator, it works w/o mag or GPS.
+	param set SYS_MC_EST_GROUP 3
+	param set ATT_ACC_COMP 0
+	param set ATT_W_ACC 0.4000
+	param set ATT_W_GYRO_BIAS 0.0000
+
+	param set SYS_HAS_MAG 0
+	
+	param set DSHOT_CONFIG 600
+	
+	param set BAT_V_CHARGED 4.20
+	param set BAT_N_CELLS 4
+	param set BAT_V_DIV 10.133
+	
+	param set SENS_BOARD_ROT 10
```

Comment:
Why is this a board default?

Reply:
I'm no PX4 coder, yet, but one of the gyros is aligned at 45 degrees to the board itself so perhaps @Igor-Misic did it for that reason.

Reply:
`SENS_BOARD_ROT` is for configuring how you mount the board. So we'll likely want to leave this parameter alone (user configurable) and handle starting the 2 x icm20602 with the rotations relative to the board.

Something like (I didn't check the specific bus numbers or rotation)
``` Console
icm20602 -s -b 1 start      # SPI 1
icm20602 -s -b 2 -R 1 start # SPI 2 rotation 45 degrees
```

Reply:
Daniel thanks for pointing on this. This definitely needs some tidying up.